src/restic/node.go

```diff
@@ -496,9 +568,35 @@ func (node *Node) fillExtra(path string, fi os.FileInfo) error {
 		err = errors.Errorf("invalid node type %q", node.Type)
 	}
 
+	switch runtime.GOOS {
+	case "windows", "openbsd":
+		return err
+	}
+	if err = node.fillAcls(path); err != nil {
+		return err
+	}
+
 	return err
 }
 
+func (node *Node) fillAcls(path string) error {
+	xattrs, e := Listxattr(path)
+	// ignore paths for which no acl can be obtained silently
+	if e == nil {
+		node.Xattrs = make([]Xattr, len(xattrs))
+		for i, attr := range xattrs {
+			attrVal, err := Getxattr(path, attr)
+			if err != nil {
+				errors.Errorf("can not obtain extended attribute %v for %v:\n", attr, path)
```

Comment:
This line makes no sense, I guess you'd like to remove the next line and replace this one with:
```go
return errors.Errorf("can not obtain extended attribute %v for %v:\n", attr, path)
```